modules/consensus/validtransaction.go

```diff
@@ -208,11 +208,11 @@ func (s *State) validTransaction(t types.Transaction) error {
 	if err != nil {
 		return err
 	}
-	err = s.validFileContractRevisions(t)
+	err = s.validStorageProofs(t)
 	if err != nil {
 		return err
 	}
-	err = s.validStorageProofs(t)
+	err = s.validFileContractRevisions(t)
```

Comment:
More consistent with the confirmation steps found in `validUnconfirmedTransaction()` in transactionpool/valid.go

Hmm, in hindsight this may have more impact than I considered. We can reject it.